rentals.rb

```diff
@@ -0,0 +1,18 @@
+require './person'
+require './book'
+
+class Rentals
+  attr_reader :person, :book
```

Comment:
Accordance to (A rental belongs to a book and also belongs to a person):
- [ ] Setters for @book and @person instance variables should be present